cmd/registry/controller/list.go

```diff
@@ -14,6 +14,7 @@ type listingClient interface {
 	ListAPIs(context.Context, names.Api, string, core.ApiHandler) error
 	ListVersions(context.Context, names.Version, string, core.VersionHandler) error
 	ListSpecs(context.Context, names.Spec, string, core.SpecHandler) error
+	ListSpecRevisions(context.Context, names.SpecRevision, string, core.SpecHandler) error
```

Comment:
Are we using the ListSpecRevisions anywhere? I think Revision support should be implicitly added to the existing Specs support that we have in the controller.